warrant/aws_srp.py

```diff
@@ -90,6 +91,17 @@ def calculate_u(big_a, big_b):
     return hex_to_long(u_hex_hash)
 
 
+def current_time():
```

Comment:
Maybe it's better to rename to `cognito_timestamp` because the function doesn't return current time?